Makefile

```diff
@@ -1140,7 +1140,7 @@ pkg-upload: pkg
 	scp -p $(TARNAME).pkg $(STAGINGSERVER):nodejs/$(DISTTYPEDIR)/$(FULLVERSION)/$(TARNAME).pkg
 	ssh $(STAGINGSERVER) "touch nodejs/$(DISTTYPEDIR)/$(FULLVERSION)/$(TARNAME).pkg.done"
 
-$(TARBALL): release-only doc-only
+$(TARBALL): release-only doc-only test/common/knownGlobals.json
```

Comment:
This target will also probably need to copy in the generated `test/common/knownGlobals.json` as the tarball is prepared in `$(TARNAME)/` from a fresh `git checkout-index`.